graphql/resolvers.go

```diff
@@ -68,6 +75,46 @@ func (r *hostResolver) Elapsed(ctx context.Context, obj *restModel.APIHost) (*ti
 	return obj.RunningTask.StartTime, nil
 }
 
+func (r *hostResolver) Volumes(ctx context.Context, obj *restModel.APIHost) ([]*restModel.APIVolume, error) {
+	volumes := make([]*restModel.APIVolume, 0, len(obj.AttachedVolumeIDs))
+	for _, volId := range obj.AttachedVolumeIDs {
+		volume, err := r.sc.FindVolumeById(volId)
+		if err != nil {
+			return volumes, InternalServerError.Send(ctx, fmt.Sprintf("Error getting volume %s", volId))
+		}
+		if volume == nil {
+			return volumes, ResourceNotFound.Send(ctx, fmt.Sprintf("Unable to find volume %s", volId))
```

Comment:
I seem to be hitting this conditional. 
![image](https://user-images.githubusercontent.com/4605522/93269057-4b53a600-f77c-11ea-99e1-b57434db6a36.png)

Reply:
I think it's okay that this condition returns an error since it would be unexpected for a host to have a `VolumeAttachment` (https://github.com/evergreen-ci/evergreen/blob/master/model/host/host.go#L112) where the volume id doesn't exist in the volumes collection